spec/Subject-spec.ts

```diff
@@ -345,33 +345,35 @@ describe('Subject', () => {
     subscription2.unsubscribe();
     subject.unsubscribe();
 
-    expect(() => {
-      subject.subscribe(
-        { next: function (x) {
+    
+    const subscription = subject.subscribe(
+      { 
+        next: function (x) {
           results3.push(x);
-        }, error: function (err) {
+        }, 
+        error: function (err) {
           expect(false).to.equal('should not throw error: ' + err.toString());
-        } }
-      );
-    }).to.throw(ObjectUnsubscribedError);
+        }
+      }
+    );
+
+    expect(subscription.closed).to.be.true;
 
     expect(results1).to.deep.equal([1, 2, 3, 4, 5]);
     expect(results2).to.deep.equal([3, 4, 5]);
     expect(results3).to.deep.equal([]);
   });
 
-  it('should not allow values to be nexted after it is unsubscribed', (done) => {
+  it('should not allow values to be nexted after it is unsubscribed', () => {
     const subject = new Subject<string>();
-    const expected = ['foo'];
-
-    subject.subscribe(function (x) {
-      expect(x).to.equal(expected.shift());
-    });
+    const results: any[] = []
```

Comment:
If possible, please turn Git hooks on so that Prettier formats this (missing `;`). The issue is present in multiple places.

Reply:
Do we happen to have a task for this?  I'll try to do this when I have time.

Reply:
We have git hooks enabled, contributors just have to perform `git commit` without `--no-verify` option.

Reply:
The specs don't have prettier running on all of them because it messes up the marble diagrams when the test scheduler is being used "the old way" (not using `TestScheduler#run()`). I'm not sure if everything has been ported yet. But I wouldn't want to include a massive formatting change in the same commit (or even PR) as a material change to the code. It would need to be done in a follow up.